docker/docker_linux.go

```diff
@@ -79,12 +80,22 @@ func GetContainerStatsByPID() (map[int32]ContainerStat, error) {
 					if err != nil {
 						continue
 					}
+					memstat, err := CgroupMemDocker(dockerStat.ContainerID)
+					if err != nil {
+						continue
+					}
+					cpuLimit, err := CgroupCPULimitDocker(dockerStat.ContainerID)
+					if err != nil {
+						continue
+					}
 
 					containerStat := ContainerStat{
-						Type:  "Docker",
-						Name:  dockerStat.Name,
-						ID:    dockerStat.ContainerID,
-						Image: dockerStat.Image,
+						Type:     "Docker",
+						Name:     dockerStat.Name,
+						ID:       dockerStat.ContainerID,
+						Image:    dockerStat.Image,
+						MemStat:  memstat,
```

Comment:
I was planning to extract out the limit on our agent code, but yea, no other stats are used so it's totally find to just record that here.